README.md

```diff
@@ -1,35 +1,466 @@
+=
 
 Requirements
-------------
+-
+Swrve supports Unity version 4.0.0 or later on Android and iOS only. **Swrve Unity SDK is not currently supported on Unity Web Player or Windows 8.**
+
 ### Unity 4.0.0+
 The SDK needs a Unity version higher than 4.0.0 to be able to compile.
 
 ### Unity 5+ Personal or Unity 4.0.0+ Pro
 Some of the features like push notifications require the use of native plugins. Native plugins are now available in Unity Personal 5+ or previvous versions of Unity Pro.
 
+Installation Instructions
+=
+1. Download the [latest release](https://github.com/Swrve/swrve-unity-sdk/releases/latest)
+
+2. Unzip the Unity SDK, import the `Swrve.unityPackage` into your project and add the `SwrvePrefab` component to your scene.
+
+  ![Import swrve.unityPackage](/docs/images/name12.jpg)
+
+  ![Add Prefab to Scene](/docs/images/name13.jpg)
+
+3. Initialize the Swrve Component
+
+  There are two methods to initialize the SDK component. Automatic initialization and manual initialization. In almost all cases, you should you manual initialization and ensure that the "initialise on start" property is not set in the Swrve prefab.
+
+  Set the `app_id` and `api_key` fields of the component to your Swrve App ID and Swrve API Key.
+
+  ```
+  // By default, Swrve stores all customer data and content in our US data center. 
+  //SwrveComponent.Instance.Config.SelectedStack = Swrve.Stack.EU;
+
+  SwrveComponent.Instance.Init(<app_id>, "<api_key>");
+  ```
+
+  You can now compile and run your project in Unity.
+
+
+4. Go to Edit > Project Settings > Script Execution Order and move `SwrveComponent` to a higher priority in the list. This prevents you from getting a `NullReferenceException` that can occur if Swrve is called before the SDK has finished initializing.
+
+### Sending Events ###
+
+```
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event");
+```
+
+When creating custom events, do not use the `swrve.*` or `Swrve.*` namespace for your own events. This is reserved for Swrve use only. Custom event names beginning with `Swrve.` are restricted and cannot be sent.
+
+An event payload can be added and sent with every event. This allows for more detailed reporting around events and funnels. The associated payload should be a dictionary of key/value pairs; it is restricted to string and integer keys and values. There is a maximum cardinality of 500 key-value pairs for this payload per event. This parameter is optional.
+
+```
+Dictionary<string,string> payload = new Dictionary<string,string>() {
+  {"key1", "value1"},
+  {"key2", "value2"}
+};
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event", payload);
+```
+
+### Send User Properties ###
+
+Assign user properties to send the status of the user. For example create a custom user property called `premium`, and then target non-premium users and premium users in the dashboard..
+
+```
+Dictionary<string, string> attributes = new Dictionary<string, string>();
```

Comment:
In the REST API they are called attributes. We can change the customer-facing name tho if needed to 'properties'
http://docs.swrve.com/developer-documentation/api-guides/swrve-events-api-guide/